src/widgets/Jitsi.ts

```diff
@@ -92,7 +92,7 @@ export class Jitsi {
         const parsed = new URL(url);
         if (parsed.hostname !== this.preferredDomain) return null; // invalid
         return {
-            conferenceId: parsed.pathname,
+            conferenceId: parsed.pathname.substr(1),
```

Comment:
Good catch! Can we get this to use `substring(1)` instead please, and a comment to say it's excluding the leading slash?

Reply:
Done with latest commit. Thanks for the feedback!